web/event_handler.go

```diff
@@ -55,7 +55,11 @@ func (fh *eventHandler) start() chan<- stopEvent {
 			select {
 			case e = <-fh.eventQueue:
 				metrics.Mark(fmt.Sprintf("events.handler.%d", fh.id))
-				metrics.UpdateGauge("events.queue.len", int64(len(fh.eventQueue)))
+
+				ql := int64(len(fh.eventQueue))
```

Comment:
Yeah, looks like the whole go world likes to use those silly one or two letter names, just tried to follow ;) Will change